css/SettingsSection.scss

```diff
@@ -26,22 +26,57 @@
 	.settings-section {
 		display: flex;
 		align-items: center;
-		margin-top: 5px;
+	}
+}
+
+.new-addressbook-entry {
+	display: flex;
+	align-items: center;
+	.icon-add {
+		width: 44px;
+		height: 44px;
 	}
 }
 
 #new-addressbook-form {
 	display: flex;
+	width: calc(100% - 44px);
```

Comment:
Not a fan of specifying width en elements that do not need to be constrained. Can't the browser guess that width from the parent ?

Reply:
that doesn't work unfortunately.